### PR TITLE
feat(blog): add form-js@1 blog post

### LINKED ISSUE #162: Blog about form-js@1.0.0
### What should we do?

We should contribute a new blog post for the recent [form-js major release](https://github.com/camunda/team-hto/issues/261).

### Why should we do it?

Let our community know about the cool new features we contributed.

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #162
Depends on https://github.com/camunda/team-hto/issues/372

Note: This is a draft, as `form-js@1.5` is yet to be released (including [dynamic lists](https://github.com/bpmn-io/form-js/pull/808)). Feel free to provide early feedback ❤️

### CODE REVIEW

File: src/blog/2023/005-form-js-1.x.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  <img src="{{ assets }}/attachments/blog/2023/005-spacer.png">
+
+  <p class="caption">
+    Use spacers to align form components to the right.
+  </p>
+</div>
+
+## New data binding capabilities
+
+Data binding receives a significant upgrade, expanding the possibilities of providing nested form keys.
```
Comment (Skaiir) [inline src/blog/2023/005-form-js-1.x.md:61]: I don't think this sentence makes sense. Worth simplifying / correcting a bit.

File: src/blog/2023/005-form-js-1.x.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-nested-key.png">
+
+  <p class="caption">
+    Use nested form keys to bind structured data.
+  </p>
+</div>
+
+Binding groups to a data path enables the provision of additional context and enhances the efficiency of organizing form data.
```
Comment (Skaiir) [inline src/blog/2023/005-form-js-1.x.md:71]: This is kind of weirdly phrased too.

File: src/blog/2023/005-form-js-1.x.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-group-path.png">
+
+  <p class="caption">
+    Specify a data path to a group to provide more context.
+  </p>
+</div>
+
+Without manual work, dynamic lists enable the effortless generation of repetitive sets of form components, such as data tables or ordered lists.
```
Comment (Skaiir) [inline src/blog/2023/005-form-js-1.x.md:81]: This one also deserves a little rewrite.

Also, I believe we should mention that we generate magic variables in the feel contexts now that we can use to access data within the same repetition, I can write something for that.

File: src/blog/2023/005-form-js-1.x.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  <p class="caption">
+    Dynamic lists allow to model repetitive lists of form components.
+  </p>
+</div>
+
+## Controllable and reusable forms via expression properties
+
+By leveraging FEEL - the expression language [standardized by the OMG](http://www.omg.org/spec/DMN/) - and [FEEL templating](https://github.com/bpmn-io/feelers), you can streamline the development process and ensure consistency across a wide range of forms, making complex form creation more efficient and reusable.
+
+These releases let you dynamically control various form properties, including read-only settings, labels, descriptions, multi-select options, and minimum and maximum validation.
```
Comment (Skaiir) [inline src/blog/2023/005-form-js-1.x.md:95]: Maybe something like "As part of these releases and moving forward, we are allowing you to define more and more properties via FEEL...", or something along those lines.

**Review by Skaiir — CHANGES_REQUESTED** [review]

File: src/blog/2023/005-form-js-1.x.md (hunk 3 above)
Comment (Skaiir) [reply src/blog/2023/005-form-js-1.x.md:81]: Or if we don't want to go too deep in the technical, we can link a future documentation page "Learn to work with dynamic lists". Although I'm not sure how okay we are with linking the Camunda docs from the bpmn-io forum.

Comment (pinussilvestrus) [reply src/blog/2023/005-form-js-1.x.md:81]: I believe it's okay to cross-reference Camunda documentation when it provides helpful additions 👍

Comment (nikku) [reply src/blog/2023/005-form-js-1.x.md:81]: What we could also link is example diagrams that show certain behaviors in use. I also consider to replace the stock form-js viewer in the demo with the playground; maybe we can even link examples there?

**Comment by pinussilvestrus** [comment]
We just decided [not to release dynamic lists with v1.4](https://github.com/camunda/team-hto/issues/372#issuecomment-1786842749) as we are currently blocked with that (FTO).

Still, this blog post will include dynamic lists as it fits into the theme of complex forms. I will update the blog post once we are ready with v1.5.

**Comment by nikku** [comment]
@Skaiir @vsgoulart Let's revisit this once the respective form-js library is released. It is a good knowledge source.

**Comment by barmac** [comment]
I rebased this on top of `main`.

**Comment by barmac** [comment]
I applied changes based on Grammarly's feedback. I believe we are ready to publish this blog post.

**Review by barmac — APPROVED** [review]

**Comment by barmac** [comment]
Great work @pinussilvestrus and @Skaiir!

**Comment by barmac** [comment]
It works fine on staging: https://stage.bpmn.io/blog/posts/2023-form-js-1.5-recent-developments

Let's release it.

**Pull request merged**